composer.json

```diff
@@ -2,17 +2,20 @@
     "name": "magento/magento-cloud-patches",
     "description": "Provides critical fixes for Magento 2 Enterprise Edition",
     "type": "magento2-component",
-    "version": "1.0.4",
+    "version": "dev-MC-34580",
```

Comment:
`"version": "dev-MC-34580"` ? it must be 1.0.4 or 1.0.5

Reply:
:) Sure
composer.json will be updated(need to upload magento/quality-patches on repo.magento.com and add some logic to .travis.yml) , please skip it for now

Reply:
composer.json has been updated, please review